### PR TITLE
[plg_authentication_cookie] Fallback to integers

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes
Params are integers but fallback values were set to strings.
Some CS (line length and camelCase).

### Testing Instructions
Code review.

### Documentation Changes Required
No.

### CODE REVIEW

File: plugins/authentication/cookie/cookie.php [hunk 1]
```diff
@@ -294,8 +295,8 @@ public function onUserAfterLogin($options)
 		}
 
 		// Get the parameter values
-		$lifetime = $this->params->get('cookie_lifetime', '60') * 24 * 60 * 60;
-		$length   = $this->params->get('key_length', '16');
+		$lifetime = $this->params->get('cookie_lifetime', 60) * 24 * 60 * 60;
```
Comment (Quy) [inline plugins/authentication/cookie/cookie.php:298]: The value will be a string if not fallback so it would be better to convert with `(int)`.

Comment (SharkyKZ) [reply plugins/authentication/cookie/cookie.php:298]: It will be an integer because the field in manifest file has integer filter applied.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c332c0ed071b6c10569bf4d70a69b9b69499797d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20338">issues.joomla.org/tracker/joomla-cms/20338</a>.</sub>

**Comment by carlitorweb** [comment]
I have tested this item :white_check_mark: successfully on c332c0ed071b6c10569bf4d70a69b9b69499797d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20338">issues.joomla.org/tracker/joomla-cms/20338</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20338">issues.joomla.org/tracker/joomla-cms/20338</a>.</sub>

**Pull request merged**